packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -703,6 +703,37 @@ There are two higher-level constructs available which include a load balancer fo
 - `LoadBalancedFargateService`
 - `LoadBalancedEc2Service`
 
+### Import existing services
+
+`Ec2Service` and `FargateService` provide methods to import existing EC2/Fargate services.
+The ARN of the existing service has to be specified to import the service.
+
+Since AWS has changed the [ARN format for ECS](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-account-settings.html#ecs-resource-ids), 
+feature flag `@aws-cdk/aws-ecs:arnFormatIncludesClusterName` must be enabled to use the new ARN format.
+The feature flag changes behavior for the entire CDK project. Therefore it is not possible to mix the old and the new format in one CDK project.
```

Comment:
This is not strictly true, but we generally recommend to not do this inline and have zero (I think?) instructions on how to do it so I don't think I mind this note here.